src/lib/components/multiDropzone/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Accept } from "react-dropzone";
+
+export type UploadStatus = 'UPLOADING' | 'COMPLETE' | 'ERROR';
+
+export type FileType =
```

Comment:
If you define this union type based on const string arrays like this:
```
export const DOCUMENT_FILES = ['doc', 'docx', 'pdf'] as const;
export const IMAGE_FILES = ['heic', 'bmp', 'jpeg', 'jpg', 'png'] as const;

export const FILE_TYPES = [...DOCUMENT_FILES, ...IMAGE_FILES] as const;
export type FileType = typeof FILE_TYPES[number];
```
you can then reuse those arrays later in your functions like `getFormattedAcceptObject`. This way you can be sure that your exported type matches the accepted file extensions. 
This means that all you need to do to add more supported files in the future, is add them to either `IMAGE_FILES` or `DOCUMENT_FILES` and everything will work out of the box with no further changes required.

Reply:
Sadly `getFormattedAcceptObject`/`accept` needs to have extensions prefixed with a `.` so I can't reuse it 😢

Reply:
I like @kimkwanka's idea, I don't see a problem in attaching the`.` to the filename in order to be correctly formatted.
You could also use the same array on the `formatAcceptFileList`.

What do you think?

Reply:
Just add a dot :D

Reply:
I'll update, I was removing the dots on another place anyway, I can add on the other cases instead 😄